tests/tree_testing.py

```diff
@@ -116,45 +119,46 @@ def check_prefix(path):
                 del self.paths[old_path]
                 self.nodes[nid].append(new_path)
                 self.paths[new_path] = nid
-    
-    def update(self,nid,path):
-        #Because of the asynchronousness of update, this test
-        #doesn't work anymore
+
+    def update(self, nid, path):
+        # Because of the asynchronousness of update, this test
+        # doesn't work anymore
         pass
-##        self.tree.flush()
-#        self.trace += "updating %s in path %s\n" %(nid,str(path))
-#        error = "updating node %s for path %s\n" %(nid,str(path))
-#        if not self.nodes.has_key(nid):
-#            error += "%s is not in nodes !\n" %nid
-#            error += self.print_tree()
-#            raise Exception(error)
-#        #Nothing to do, we just update.
-#        for p in self.nodes[nid]:
-#            if self.paths[p] != nid:
-#                raise Exception('Mismatching path for %s'%nid)
-#        if not self.paths.has_key(path):
-#            error += '%s is not in stored paths (node %s)\n'%(str(path),nid)
-#            error += self.print_tree()
-#            raise Exception(error)
-#        n = self.paths[path]
-#        if path not in self.nodes[n] or n != nid:
-#            raise Exception('Mismatching node for path %s'%str(p))
-            
-    def reordered(self,nid,path,neworder):
-        print("reordering")
-        self.trace += "reordering children of %s (%s) : %s\n" %(nid,str(path),neworder)
-        self.trace += "VR is %s\n" %self.tree.node_all_children()
+        # self.tree.flush()
+        # self.trace += "updating %s in path %s\n" % (nid, str(path))
+        # error = "updating node %s for path %s\n" % (nid, str(path))
+        # if not self.nodes.has_key(nid):
+        #     error += "%s is not in nodes !\n" %nid
+        #     error += self.print_tree()
+        #     raise Exception(error)
+        # # Nothing to do, we just update.
+        # for p in self.nodes[nid]:
+        #     if self.paths[p] != nid:
+        #         raise Exception('Mismatching path for %s'%nid)
+        # if not self.paths.has_key(path):
+        #     error += '%s is not in stored paths (node %s)\n'% (
+        #        str(path), nid)
+        #     error += self.print_tree()
+        #     raise Exception(error)
+        # n = self.paths[path]
+        # if path not in self.nodes[n] or n != nid:
+        #     raise Exception('Mismatching node for path %s'%str(p))
```

Comment:
Leaving commented out code is not advised... any reason for keeping this section?